### DISCUSSION TITLE
docs/v1.10.x/start/prepare/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2527] [Announcements]
# docs/v1.10.x/start/prepare/

https://kubeovn.github.io/docs/v1.10.x/start/prepare/

**Comment by xinsir88** [comment]
请教大佬，我安装ovn后，有几个问题。
1. kube-ovn-pinger DS资源没有pod创建。
2.kube-ovn-controller Deployment资源始终不能成功创建。
3.kube-ovn-cni Deployment资源始终不能成功创建。
